dnf5daemon-client/commands/repoquery/repoquery.cpp

```diff
@@ -219,16 +219,14 @@ void RepoqueryCommand::run() {
 
     int in_fd = pipefd[0];
 
-    const size_t pipe_buffer_size = fcntl(in_fd, F_GETPIPE_SZ);
+    const auto pipe_buffer_size = static_cast<size_t>(fcntl(in_fd, F_GETPIPE_SZ));
```

Comment:
We should handle situation when `fcntl` fails and return -1 and add some reasonable fallback value - 4096?

Reply:
@m-blaha Good idea. Moreover, F_GETPIPE_SZ is Linux specific, so a fallback would be useful.

Reply:
Maybe we can just go with a constant value. Looks like on my system the default value is 65536 and do not try to detect the size.

Reply:
Anyway, it's out of this PR scope, I filed a new issue for it - https://github.com/rpm-software-management/dnf5/issues/1419